.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,29 @@
+name: Feature Request
+description: Suggest a new feature for freud
+body:
+  - type: textarea
+    attributes:
+      label: Problem
```

Comment:
Calling this "Problem" sounds odd, but given the context and that the next section is the "Description" I'm not sure what alternative to suggest. We can move forward as is, just making this comment in case someone else comes up with a better alternative.

Reply:
I mostly tried to copy the formatting from the previous issue templates, where the first section was something like "What is the problem this feature is trying to solve?". Feel free to suggest another way of wording the title.

Reply:
There is also a "Description" section below, though. Otherwise I would also have suggested "Description" here.

As I said, I don't have a great alternative to suggest so if neither of you can come up with one it's fine with me if to move forward as is.

Reply:
I may just reformat the first few sections here, we don't have to strictly follow the wording/section layout that was used before.